agent/app/agent_capability.go

```diff
@@ -297,6 +302,7 @@ func (agent *ecsAgent) capabilities() ([]*ecs.Attribute, error) {
 		}
 		capabilities = removeAttributesByNames(capabilities, externalUnsupportedCapabilities)
 	}
+	capabilities = agent.appendGuardDutyCapabilities(capabilities)
```

Comment:
Could we add a comment `// add GuardDuty capabilities if applicable` for this line?

Reply:
Added the comment.